### DISCUSSION TITLE
Regarding the uses of SetIdentityFunction in GeoMechanicsApplication

### DISCUSSION THREAD

**Discussion by sunethwarna** [KratosMultiphysics/Kratos#12108] [Q&A] [answered]
Hi @KratosMultiphysics/geomechanics,

I saw that you are using `SetIdentityFunction` in the following place.
https://github.com/KratosMultiphysics/Kratos/blob/50ebdf1b35f5ebb32f7ee6e5f97c611671c699ea/applications/GeoMechanicsApplication/custom_workflows/dgeoflow.h#L71

I have not seen or I could not find the use cases of `result_type` in the rest of the code.
- Is there a specific reason to have that typedef?
- Would it be possible to use the typedefs from the Kratos core?

As we are working on changing the `SetIdentityFunction` to be more robust (see #12087 ), We would like to ask whether you can rely on the core typedefs rather than using the `result_type` as used in the code snippet?

Thanks :)

**Comment by avdg81** [comment, marked as answer]
Hi @sunethwarna, we have checked the alias that you've mentioned, and, like you, we couldn't find any usages. So we have removed it from our code. See #12110. Thanks for making us aware of this opportunity to improve the code.
